config/default.yaml

```diff
@@ -119,6 +119,15 @@ executor:
 
         # Launcher container tag to use
         launchVersion: stable
+    jenkins:
+      # Configuration of Jenkins
+      jenkins:
```

Comment:
Can we make this:
```
jenkins:
     options:
         host: ...
         port: ....
         ... 
```
to be consistent with other executors?

This would also affect our Queue work. Making it consistent will help a lot.